src/sortStudents.ts

```diff
@@ -1,16 +1,77 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copyStudents = [...students];
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  if (order === 'asc') {
+    switch (sortBy) {
+      case SortType.Name:
+        return copyStudents.sort((a, b) => a.name.localeCompare(b.name));
+      case SortType.Surname:
+        return copyStudents.sort((a, b) => a.surname.localeCompare(b.surname));
```

Comment:
```suggestion
      case SortType.Name:
      case SortType.Surname:
        return copyStudents.sort((a, b) => a[sortBy].localeCompare(b[sortBy]));
```
you are doing the same but with different keys
add values to `SortType` enum properties that match `Student` object properties to be able use it as key